### ISSUE TITLE
Kapacitor crashed

### ISSUE THREAD

**Issue by m4ce** [influxdata/kapacitor#1145]
I guess it shouldn't crash? I am using 1.2.0-rc1.

```
[alert-system-netif_duplex:last4] 2017/01/24 09:44:01 E! failed to aggregate batch: field duplex missing from point cannot aggregate
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x607393]
goroutine 271 [running]:
panic(0xb6e740, 0xc42000c070)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/influxdata/kapacitor/vendor/github.com/influxdata/influxdb/influxql.(*IntegerFuncReducer).Emit(0xc420438400, 0xc420cfc610, 0x49a1a4, 0x4969f5)
        /root/go/src/github.com/influxdata/kapacitor/vendor/github.com/influxdata/influxdb/influxql/functions.gen.go:562 +0x43
github.com/influxdata/kapacitor.(*integerPointEmitter).EmitPoint(0xc420a588f0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /root/go/src/github.com/influxdata/kapacitor/influxql.gen.go:403 +0x7c
github.com/influxdata/kapacitor.(*InfluxQLNode).emit(0xc4201ebc00, 0x10ae240, 0xc420a588c0, 0x2, 0xc420438430)
        /root/go/src/github.com/influxdata/kapacitor/influxql.go:231 +0x1f8
github.com/influxdata/kapacitor.(*InfluxQLNode).runBatchInfluxQL(0xc4201ebc00, 0x0, 0x0)
        /root/go/src/github.com/influxdata/kapacitor/influxql.go:207 +0xe93
github.com/influxdata/kapacitor.(*InfluxQLNode).runInfluxQLs(0xc4201ebc00, 0x0, 0x0, 0x0, 0xc420025758, 0xc420025768)
        /root/go/src/github.com/influxdata/kapacitor/influxql.go:42 +0x4c
github.com/influxdata/kapacitor.(*InfluxQLNode).(github.com/influxdata/kapacitor.runInfluxQLs)-fm(0x0, 0x0, 0x0, 0xc420025780, 0xc42028ed80)
        /root/go/src/github.com/influxdata/kapacitor/influxql.go:33 +0x48
github.com/influxdata/kapacitor.(*node).start.func1(0xc4201ebc00, 0x0, 0x0, 0x0)
        /root/go/src/github.com/influxdata/kapacitor/node.go:130 +0x8e
created by github.com/influxdata/kapacitor.(*node).start
        /root/go/src/github.com/influxdata/kapacitor/node.go:131 +0x5d
```

After I added the default() node, it worked.

**Comment by nathanielc** [comment]
@m4ce Can you share your TICKscript?

**Comment by nathanielc** [comment]
@m4ce This was fixed as per #872. Upgrading to 1.2.0 will fix the issue.

**Comment by m4ce** [comment]
Thanks @nathanielc, I'll do that right away.
